olive/passes/onnx/optimum_conversion.py

```diff
@@ -21,10 +21,29 @@ class OptimumConversion(Pass):
 
     @staticmethod
     def _default_config(accelerator_spec: AcceleratorSpec) -> Dict[str, PassConfigParam]:
+        device = "cuda" if accelerator_spec.accelerator_type == "gpu" else "cpu"
         config = {
             "target_opset": PassConfigParam(
                 type_=int, default_value=14, description="The version of the default (ai.onnx) opset to target."
-            )
+            ),
+            "fp16": PassConfigParam(
```

Comment:
We could add a search point validation so that `fp16` can only be true if `"cuda" in device`.

Reply:
Done.